andela/ah-backend-realers
# #166187853 Implement descriptive validation erros messages

#### What does this PR do?
- Users should receive a descriptive error message when they enter invalid user parameters in an attempt to signup.

#### Description of Task to be completed?
- User should get descriptive validation errors messages on registration when they enter invalid user parameters when signing up.

#### How should this be manually tested?
Run git fetch
Then check out `ft-signup-validations-166187853`
Then using postman run this `http://127.0.0.1:8000/api/users/ `providing wrong data


#### What are the relevant pivotal tracker stories?
[Finishes #166187853]

#### Screenshots (if appropriate)?
<img width="957" alt="Screenshot 2019-06-10 at 1 14 08 PM" src="https://user-images.githubusercontent.com/43774983/59188999-acf09200-8b81-11e9-93ca-9f3dc211e332.png">
